jdbc/src/test/java/io/kestra/jdbc/runner/JdbcHeartbeatTest.java

```diff
@@ -175,10 +178,17 @@ void triggerResubmit() throws Exception {
         AtomicReference<WorkerTriggerResult> workerTriggerResult = new AtomicReference<>(null);
         workerTriggerResultQueue.receive(either -> {
             workerTriggerResult.set(either.getLeft());
+            countDownLatch.countDown();
```

Comment:
This count down latch seems to be unused now

Reply:
It's used L197 to await till a triggerResult is received to ensure that the resubmit worked properly and that the second worker took the task